### PR TITLE
bump rancher-webhook to 0.1.200

### PR DESCRIPTION
**Pull request by jiaqiluo**
issue: 
https://github.com/rancher/rancher/issues/34453
https://github.com/rancher/rancher/issues/34320

### CODE REVIEW

File: package/Dockerfile [hunk 1]
```diff
@@ -40,7 +40,7 @@ ENV KUSTOMIZE_VERSION v3.5.4
 # System charts minimal version
 ENV CATTLE_FLEET_MIN_VERSION=0.3.500
 ENV CATTLE_RANCHER_OPERATOR_MIN_VERSION=0.1.400
-ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.000
+ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.200-rc1
```
Comment (StrongMonkey) [inline package/Dockerfile:43]: You will just need to update to 0.1.200. Any version that is below 0.1.200(including rcs) will be installed/upgrade.

Comment (jiaqiluo) [reply package/Dockerfile]: fixed!

**Review by StrongMonkey — APPROVED** [review]

**Review by aiyengar2 — APPROVED** [review]
LGTM

File: package/Dockerfile [hunk 2]
```diff
@@ -40,7 +40,7 @@ ENV KUSTOMIZE_VERSION v3.5.4
 # System charts minimal version
 ENV CATTLE_FLEET_MIN_VERSION=0.3.500
 ENV CATTLE_RANCHER_OPERATOR_MIN_VERSION=0.1.400
-ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.000
+ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.200
```
Comment (cbron) [inline package/Dockerfile:43]: Same question as here: https://github.com/rancher/charts/pull/1458/files#r704828161
Are we confident all of those changes can be backported. We probably should have branched the webhook.

Comment (jiaqiluo) [reply package/Dockerfile:43]: The changes include adding the support for hardened cluster and a pre-delete-hook for cleaning up `mutatingwebhookconfigurations`, both of them can be backported. 

We have branched the app,   `0.1.200` is cut from the `release/v0.1` branch for rancher 2.5

**Review by cbron — APPROVED** [review]
Already in the new charts issue. Lgtm.

**Pull request merged**